tests/v2/validation/rbac/rbac_test.go

```diff
@@ -335,6 +335,67 @@ func (rb *RBTestSuite) ValidateAddCMAsProjectOwner() {
 
 }
 
+func (rb *RBTestSuite) ValidateAddPOsAsProjectOwner() {
+	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	rb.standardUserCOProject = createProjectAsCO
+
+	log.Info("Additional Testcase4 - Validating if Project Owner can add another Project Owner ")
+	//Additional test4 validate if member with role Project Owner can add a project owner
+	errUserRole := users.AddProjectMember(rb.standardUserClient, rb.standardUserCOProject, rb.additionalUser, roleProjectOwner)
+	require.NoError(rb.T(), errUserRole)
+	rb.additionalUserClient, err = rb.additionalUserClient.ReLogin()
+
+	additionalUser1, err := createUser(rb.client, standardUser)
+	additionalUserClient1, err := rb.client.AsUser(additionalUser1)
```

Comment:
and once we update the name above, can we also update this to addNewUserAsPOClient ?